packages/polaris-viz/src/components/FunnelChartNext/Chart.tsx

```diff
@@ -16,18 +16,28 @@ import {
 import {FunnelChartSegment} from '../shared';
 import {ChartElements} from '../ChartElements';
 
-import {FunnelChartLabels} from './components';
+import {
+  FunnelChartLabels,
+  FunnelTooltip,
+  Tooltip,
+  TooltipWithPortal,
+} from './components';
 import {
   LABELS_HEIGHT,
   LINE_GRADIENT,
   LINE_OFFSET,
   LINE_WIDTH,
   GAP,
   SEGMENT_WIDTH_RATIO,
+  TOOLTIP_HORIZONTAL_OFFSET,
+  TOOLTIP_HEIGHT,
+  TOOLTIP_WIDTH,
 } from './constants';
+import type {FunnelChartNextProps} from './FunnelChartNext';
 
 export interface ChartProps {
   data: DataSeries[];
+  tooltipLabels: FunnelChartNextProps['tooltipLabels'];
```

Comment:
I wonder if we should make this `tooltipLabels` optional as we are not going to use them for `open` funnels?